### DISCUSSION TITLE
After updating emqx from 4.3.10 to 4.4.4 errors are coming

### DISCUSSION THREAD

**Discussion by Aravindms369** [emqx/emqx#8199] [Q&A] [answered]
emqx: 4.4.4

I have migrated my emqx from 4.3.10 to 4.4.4 for removing the e-redis client error and generic server terminating error. After migration we are receiving below errors in the logs. device operations are working fine.

**[warning]** [SYSMON] long_schedule warning: pid = <****>, info: [{timeout,366}, {in,{prim_inet,accept0,3}}, {out, {ranch_conns_sup, start_protocol,2}}], [{memory,***},{total_heap_size,**},{heap_size,**},{stack_size,9},{min_heap_size,*},{initial_call,{ranch_acceptor,loop,4}},{current_stacktrace,[{ranch_conns_sup,start_protocol,2,[{file,"ranch_conns_sup.erl"},{line,72}]},{ranch_acceptor,loop,4,[{file,"ranch_acceptor.erl"},{line,34}]}]},{registered_name,[]},{status,waiting},{message_queue_len,0},{group_leader,<**>},{priority,normal},{trap_exit,false},{reductions,104139702},{last_calls,false},{catchlevel,0},{trace,0},{suspending,[]},{sequential_trace_token,[]},{error_handler,error_handler}]

**[error]** supervisor: 'esockd_connection_sup - <***>', errorContext: connection_shutdown, reason: {ssl_error,{tls_alert,{certificate_required,"TLS server: In state wait_cert at tls_handshake_1_3.erl:1475 generated SERVER ALERT: Fatal - Certificate required\n certificate_required"}}}, offender: [{pid,<****>},{name,connection},{mfargs,{emqx_connection,start_link,[[{deflate_options,[]},{max_conn_rate,500},{active_n,100},{zone,external},{proxy_address_header,<<>>},{proxy_port_header,<<>>},{supported_subprotocols,[]}]]}}]

**[error]** supervisor: 'esockd_connection_sup - <****>', errorContext: connection_shutdown, reason: {ssl_error,{tls_alert,{unexpected_message,"TLS server: In state hello at tls_record.erl:558 generated SERVER ALERT: Fatal - Unexpected Message\n {unsupported_record_type,71}"}}}, offender: [{pid,<****>},{name,connection},{mfargs,{emqx_connection,start_link,[[{deflate_options,[]},{max_conn_rate,500},{active_n,100},{zone,external},{proxy_address_header,<<>>},{proxy_port_header,<<>>},{supported_subprotocols,[]}]]}}]

please help to resolve the same.

**Comment by zmstone** [comment, marked as answer]
answered in https://github.com/emqx/emqx/issues/8200

**Comment by Aravindms369** [comment]
Hi @zmstone

emqx version:4.4.4

I am continuously getting the below warning logs in emqx:

[warning] Ekka(AutoCluster): discovery did not succeed; retrying in 5000 ms

Please help to resolve the same.

**Reply by zmstone** [reply]
hi. this is a different topic, could you create a new thread ?

**Reply by Aravindms369** [reply]
Okay, Sure @zmstone
